preprocessing/nextclade/src/loculus_preprocessing/processing_functions.py

```diff
@@ -65,7 +88,7 @@ def format_authors(authors: str) -> bool:
             [f"{name}." if len(name) == 1 else name for name in first_name.split(" ")]
```

Comment:
Capitalize initials as well
```suggestion
            [f"{name.upper()}." if len(name) == 1 else name for name in first_name.split(" ")]
```